testsuite/integration/basic/src/test/config/arq/arquillian.xml

```diff
@@ -7,7 +7,7 @@
     <container qualifier="jboss" default="true">
         <configuration>
             <property name="jbossHome">${basedir}/target/jbossas</property>
-            <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/jbossas -Dtest.bind.address=${node0}</property>
+            <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/jbossas</property>
```

Comment:
QE uses the `node0` to configure the bind address. Nonetheless it seems the `"test.bind.address"` is not used in the testsuite, so it should be fine to remove this one from the configuration.